optimize/backend/src/main/java/io/camunda/optimize/tomcat/ExternalHomeServlet.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
This is the new servlet I mentioned in my own comment above. It serves the static files under the webapp directory (the built client, basically)